website/static/js/registrationUtils.js

```diff
@@ -450,15 +449,19 @@ var RegistrationEditor = function(urls, editorId) {
         var t = self.lastSaveTime();
         if (t) {
             return t.toGMTString();
-        } 
-        else {
+        } else {
             return 'never';
         }
     });
 
+    self.canRegister = ko.computed(function() {
+        var draft = self.draft();
```

Comment:
This can be a `pureComputed`, as it has no side effects etc. (but does depend on an observable)